float64 rate        # IMU rate [Hz]
float64 a_max       # Max accelerometer measurement [m/s^2]
float64 g_max       # Max gyroscope measurement [rad/s]
float64 sigma_g_c   # Gyro noise density [rad/s/sqrt(Hz)]
float64 sigma_a_c   # Accel noise density [m/s^s/sqrt(Hz)]
float64 sigma_gw_c  # Gyro drift noise density [rad/s^s/sqrt(Hz)]
float64 sigma_aw_c  # Accel drift noise density [m/s^2/sqrt(Hz)]
float64 sigma_bg    # Gyro bias prior [rad/s]
float64 sigma_ba    # Accel bias prior [m/s^2]
float64 g           # Gravity vector [ms^-2]
